### ISSUE TITLE
Feature: Make components accessible

### ISSUE THREAD

**Issue by dnwhte** [Formstone/Formstone#172]
A great set of components. However the lack of accessibility makes them unusable for many (especially government institutions, universities, colleges, etc).

**Comment by benplum** [comment]
Which components specifically? We've been working on accessibility, starting with the most used components.

**Comment by dnwhte** [comment]
I was looking at using the scrollbar, navigation, and tabs.

**Comment by dnwhte** [comment]
http://frend.co/ is doing a good job with accessible components. Their library is really small though. Might be worth a look.

**Comment by benplum** [comment]
I actually just found this earlier today :) as well as http://a11yproject.com/patterns/.

**Comment by benplum** [comment]
We've got an accessibility branch going, and feedback is welcome! https://github.com/Formstone/Formstone/tree/ally

**Comment by benplum** [comment]
The next release will feature a big accessibility overhaul.

**Comment by benplum** [comment]
This is up on the dev branch and will go live with 1.2.0
